### DISCUSSION TITLE
Support for dummy data for npm run dev

### DISCUSSION THREAD

**Discussion by saugatmaharjan** [resend/react-email#927] [Ideas]
### Goals

1. let users to pass dummy data from preview while developing template
2.
3.

### Background

can't use npm run dev to preview while components props are dynamic, had to create own server

### Proposal

enable users to create a .js or .json file with same name as template and load data to use as props for the template

**Comment by shellscape** [comment]
It's not documented but you can use a defaultPreviewData property on your exported template. check out the source for the example templates, there are a few.

working on a replacement for the preview/dev server that will make that a lot easier in the future

**Reply by Hainesy** [reply]
Has this changed? I can't find `defaultPreviewData` anywhere, and it doesn't work for me.

**Reply by Hainesy** [reply]
Found it's changed to `PreviewProps`
